etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
   name: pachd
   namespace: {{ .Release.Namespace }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.pachd.replicas }}
```

Comment:
at the moment there are some issues with multiple replicas of pachd. Could you remove the param from this PR and we'll try to address it soon?